packages/browser-agent-core/src/common/wrap/wrap-history.js

```diff
@@ -3,30 +3,45 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-// History pushState wrapper
-import {ee as baseEE} from '../event-emitter/contextual-ee'
-import {createWrapperWithEmitter as wfn} from './wrap-function'
+/**
+ * Wrapper for pushState and replaceState methods of window.history object
+ */
+
+import { ee as globalEE } from '../event-emitter/contextual-ee'
+import { createWrapperWithEmitter as wfn } from './wrap-function'
 import { isBrowserScope } from '../util/global-scope'
 
-const wrapped = {}
+let alreadyWrapped = false
```

Comment:
this may have consequences in the niche scenario where multiple agents are on one page that share the same bundle, is there a reason this part was changed?  It used to have a namespace for each potential agent ID to separate that

Reply:
I figured it was an unnecessary remnant. Makes sense that it would pertain to multi-agent scenarios.